.travis.yml

```diff
@@ -69,8 +69,9 @@ before_script:
     # Make sketchbook/libraries folder.
     - mkdir -p $CI_SRC_PATH/sketchbook/libraries
     - export CI_LIB_PATH=$CI_SRC_PATH/sketchbook/libraries
-    # Make rosserial_arduino in the sketchbook/libraries folder, including victoria_nav_msgs
-    - rosrun rosserial_arduino make_libraries.py $CI_LIB_PATH victoria_nav_msgs
+    # Make rosserial_arduino in the sketchbook/libraries folder,
+    # including victoria_nav_msgs and victoria_sensor_msgs
+    - rosrun rosserial_arduino make_libraries.py $CI_LIB_PATH victoria_nav_msgs,victoria_sensor_msgs
```

Comment:
Not after it worked with the comma. I always find what I am looking for in the last place I look. :-)

It works with the space though.  I'll update the file to that.  But my larger comment is that I completely left it out in the first version (I only added it to the comment), and everything still passed in travis. It should have had failed?